src/Cassandra/HostConnectionPool.cs

```diff
@@ -218,6 +218,8 @@ internal void AttemptReconnection()
                 }
                 else
                 {
+                    if (t.Exception != null)
+                        Logger.Error("Error during reconnection attempt", t.Exception);
```

Comment:
Now that I think of it, it is not necessary to include the exception in the logs again. It was already logged on `CreateConnection()`.